.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+<!-- Provide a general summary of your changes in the Title above -->
+<!-- It must be meaningful and coherent with the changes -->
+
+<!--
+    If this PR is still a Work in Progress [WIP], please open it as DRAFT.
+    Please consider if any label should be added to this PR.
+-->
+
+## Description
+<!--
+    Describe changes in detail.
+    This includes depicting the context, use case or current behavior and describe the proposed changes.
+    If several features/bug fixes are included with these changes, please consider opening separated pull requests.
+-->
+
+<!--
+    In case of bug fixes, please provide the list of supported branches where this fix should be also merged.
+    Please uncomment following line, adjusting the corresponding target branches for the backport.
+-->
+<!-- @Mergifyio backport 2.2.x 1.1.x 1.0.x -->
+
+<!-- If an issue is already opened, please uncomment next line with the corresponding issue number. -->
+<!-- Fixes #(issue) -->
+
+<!-- In case the changes are built over a previous pull request, please uncomment next line. -->
+<!-- This PR depends on #(PR) and must be merged after that one. -->
+
+## Contributor Checklist
+- [ ] Commit messages follow the project guidelines. <!-- External contributors should sign the DCO. Fast DDS developers must also refer to the internal Redmine task. -->
+- [ ] Any new/modified methods have been properly documented using Doxygen. <!-- Even internal classes, and private methods and members should be documented, not only the public API. -->
+- [ ] Tests pass locally.
+- [ ] Applicable backports have been included in the description.
```

Comment:
I think API and ABI compatibility and also quite important here. I'd also include the cross emoji that we have on Fast DDS. In fact, this list could be almost the same as the one in Fast DDS